pkg/storaged/part-tab.jsx

```diff
@@ -22,35 +22,35 @@ import { DescriptionList, DescriptionListDescription, DescriptionListGroup, Desc
 
 import cockpit from "cockpit";
 import * as utils from "./utils.js";
+import { useMock } from "hooks.js";
 
 const _ = cockpit.gettext;
 
-export class PartitionTab extends React.Component {
-    render() {
-        const block_part = this.props.client.blocks_part[this.props.block.path];
-
-        return (
-            <DescriptionList className="pf-m-horizontal-on-sm">
-                <DescriptionListGroup>
-                    <DescriptionListTerm>{_("Name")}</DescriptionListTerm>
-                    <DescriptionListDescription>{block_part.Name || "-"}</DescriptionListDescription>
-                </DescriptionListGroup>
-
-                <DescriptionListGroup>
-                    <DescriptionListTerm>{_("Size")}</DescriptionListTerm>
-                    <DescriptionListDescription>{utils.fmt_size(block_part.Size)}</DescriptionListDescription>
-                </DescriptionListGroup>
-
-                <DescriptionListGroup>
-                    <DescriptionListTerm>{_("UUID")}</DescriptionListTerm>
-                    <DescriptionListDescription>{block_part.UUID}</DescriptionListDescription>
-                </DescriptionListGroup>
-
-                <DescriptionListGroup>
-                    <DescriptionListTerm>{_("Type")}</DescriptionListTerm>
-                    <DescriptionListDescription>{block_part.Type}</DescriptionListDescription>
-                </DescriptionListGroup>
-            </DescriptionList>
-        );
-    }
-}
+export const PartitionTab = ({ client, block }) => {
+    const mock_UUID = useMock()?.part_UUID;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17654-20230302-101354-40692ee9-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)